tasks/Functions-admin/setNodeKey.js

```diff
@@ -0,0 +1,37 @@
+const { VERIFICATION_BLOCK_CONFIRMATIONS, networkConfig } = require("../../network-config")
+
+task("functions-set-node-key", "Sets the per-node public key in the Functions oracle contract")
+  .addParam("key", "Node-assigned public key")
+  .addOptionalParam("node", "Address of the node for which the public key is to be set (defaults to msg.sender)")
+  .setAction(async (taskArgs) => {
+    if (network.name === "hardhat") {
+      throw Error("This command cannot be used on a local development chain.  Specify a valid network.")
+    }
+
+    let overrides = undefined
+    if (network.name === "goerli") {
```

Comment:
We shouldn't be copy-pasting this block in every command. At least make it a shared functions.

Reply:
Is this change going to block this PR or can we merge it for now?  Changing this is going to require changing every task file in the repo.

Reply:
I haven't "requested changes" so once @pinebit approves you're good to go.